netci.groovy

```diff
@@ -2329,13 +2413,13 @@ Constants.allScenarios.each { scenario ->
                     else {
                         // Non-Windows
                         if (architecture == 'arm64') {
-                            if (scenario != 'default' && scenario != 'r2r' && scenario != 'gcstress0x3' && scenario != 'gcstress0xc') {
+                            if (scenario != 'innerloop' && scenario != 'default' && scenario != 'r2r' && scenario != 'gcstress0x3' && scenario != 'gcstress0xc') {
```

Comment:
Small savings: could substitute "!isDefaultOrInnerloop" for "scenario != 'innerloop' && scenarios != 'default'" but honestly this is borderline big enough to warrant a switch table or a table lookup.